server/odc-service/src/main/java/com/oceanbase/odc/service/task/service/StdTaskFrameworkService.java

```diff
@@ -310,8 +313,11 @@ public void handleResult(TaskResult taskResult) {
             log.warn("Job identity is not exists by id {}", taskResult.getJobIdentity().getId());
             return;
         }
-        if (je.getStatus().isTerminated() || je.getStatus() == JobStatus.CANCELING) {
-            log.warn("Job {} is finished, ignore result, currentStatus={}", je.getId(), je.getStatus());
+        if (je.getStatus() == JobStatus.CANCELING || !taskFrameworkEnabledProperties.isEnabled()) {
```

Comment:
why save log meta while  task framework not enabled

Reply:
task is running before close task framework,   when close task framework we send stop command to pod, then task will be stopped and upload log to oss,  we save the report result  about oss info